module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/SearchTabsTest.php

```diff
@@ -162,15 +283,50 @@ protected function getHelper(
             ->method('get')
             ->willReturn($solr);
 
+        $request = Request::fromString('GET / HTTP/1.1');
+        if ($filters) {
+            $queryFilters = [];
+            foreach ($filters as $key => $filter) {
+                $queryFilters[] = "$key:\"$filter\"";
+            }
+            $request->getQuery()->hiddenFilters = $queryFilters;
+        }
+
         $url = $this->getMockBuilder(Url::class)
             ->disableOriginalConstructor()
             ->getMock();
-        $searchTabsHelper = $this->getMockBuilder(SearchTabsHelper::class)
-            ->disableOriginalConstructor()
+        $searchTabsHelper = new SearchTabsHelper(
+            $resultsPM,
+            $tabConfig,
+            $tabFilterConfig,
+            $request,
+            [
+                'Solr:filtered' => 'logged-in',
+            ]
+        );
+        $ssearchTabsHelper = $this->getMockBuilder(SearchTabsHelper::class)
+            ->setConstructorArgs(
+                [
+                    $resultsPM,
+                    [],
+                    [],
+                    $request,
+                    [
+                        'Solr:filtered' => 'logged-in',
+                    ],
+                    [],
+                ]
+            )->onlyMethods(['getHiddenFilters', 'getTabConfig', 'getTabFilterConfig'])
             ->getMock();
-        $searchTabsHelper->expects($this->exactly($filtersCalls))
+        /*$searchTabsHelper->expects($this->exactly($filtersCalls))
```

Comment:
Is this commented-out code left here for a reason?

Reply:
Oops, no. Now cleaned up!